components/brave_ads/core/internal/common/challenge_bypass_ristretto/challenge_bypass_ristretto_test.cc

```diff
@@ -49,20 +48,17 @@ TEST(BraveAdsChallengeBypassRistrettoTest, ProveAndVerifyUnblindedToken) {
   EXPECT_TRUE(public_key);
 
   // Server signs a DLEQ (Discrete Log Equivalence) proof.
```

Comment:
This comment should be `// Server returns a batch DLEQ (Discrete Log Equivalence) proof.`